cli/app.go

```diff
@@ -96,6 +96,7 @@ const (
 	dataFlagParallelDownloads              = "parallel"
 	dataFlagTags                           = "tags"
 	dataFlagBboxLabels                     = "bbox-labels"
+	dataFlagDatasetIDFilter                = "dataset-filter-id"
```

Comment:
This is kind of awkward and I'm not fully sure where it should go, but since we are transferring data from dataset to another I know we need a new flag. Let me know if I should change the name